src/Microsoft.VisualStudio.AppDesigner/PropPages/PropPageUserControlBase.vb

```diff
@@ -3337,7 +3341,19 @@ NextControl:
         ''' <returns></returns>
         ''' <remarks></remarks>
         Protected Function GetProjectPath() As String
-            Return CStr(GetCommonPropertyValueNative("FullPath")) 'CONSIDER: This won't work for all project types, e.g. ASP.NET when project path is a URL
```

Comment:
BrowseObject != Project Properties, basically if you add it to ConfigurationGeneral, it should show up.